Hymns/AppDelegate.swift

```diff
@@ -1,4 +1,4 @@
-import Firebase
+import FirebaseCore
```

Comment:
Is this needed?

Reply:
So for whatever reason my XCode keeps throwing me an error about this when I use Firebase now. It gives me a red error flag but it actually does not stop the build. This started happening to me awhile back. I don't know if it was because our pods were updated or what. I can revert though. You don't get any error flags?

Reply:
yeah it doesn't cause any errors for me lol